tests/src/main/java/com/squareup/duktape/tests/OctaneActivity.java

```diff
@@ -0,0 +1,68 @@
+package com.squareup.duktape.tests;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.view.View;
+import android.view.View.OnClickListener;
+import android.widget.TextView;
+import com.squareup.duktape.Duktape;
+import java.io.IOException;
+import okio.BufferedSource;
+import okio.Okio;
+
+public final class OctaneActivity extends Activity implements Runnable {
```

Comment:
Yeah it didn't do this on my test devices. It's easy to add though, and probably good to show some indication of work being done.